Geek-Collectors-Network/geek-collectors-network
# 107 make command to populate db witih dummy data

### Summary
 - Create script to drop all tables without deleting databases
 - Automatically populate database with dummy data in development, if it doesn't already exist

### Reviewer's Notes
None of these implementations are final, I'm more looking for feedback.

### Testing
1. Update your local api .env file (use .env.example as reference)
2. Spin up resources: `make resources`
3. Start API server: `npm run -w apps/api start:dev`
4. Check that the tables are populated with dummy data
5. Drop all tables: `make clean-db`
6. Check that the databases no longer have tables
